examples/plotting_sunspots_using_stara.py

```diff
@@ -0,0 +1,99 @@
+"""
+=================================
+Plotting Sunspots using STARA
+=================================
+
+This example demonstrates the use of Sunspot Tracking And Recognition
```

Comment:
I wonder if a link to the paper is useful here.

Reply:
Indeed, for verification purposes, it's beneficial. Additionally, I think including a reference to Will's [notebook](https://gitlab.com/wtbarnes/aia-on-pleiades/-/blob/master/notebooks/tidy/finding_sunspots.ipynb), which demonstrates this algorithm using Dask arrays. This could be advantageous for those interested in conducting extensive time period analyses.